dot/digest/digest_test.go

```diff
@@ -38,7 +38,8 @@ func newTestHandler(t *testing.T) *Handler {
 	err = stateSrvc.Start()
 	require.NoError(t, err)
 
-	dh, err := NewHandler(stateSrvc.Block, stateSrvc.Epoch, stateSrvc.Grandpa)
+	logger := log.NewFromGlobal(log.AddContext("pkg", "digest"))
```

Comment:
Let's get rid of the logs during tests by default :wink: 

```suggestion
	logger := log.New(log.SetWriter(io.Discard)))
```

Reply:
I don't think we need to entirely get rid of them, we might want them at some point, maybe just set to LevelCrit?

Reply:
Same thing, you can just change `io.Discard` with `os.Stdout` :wink: